cirq-google/cirq_google/experimental/ops/coupler_pulse.py

```diff
@@ -63,12 +68,17 @@ def __init__(
             coupling_mhz: Target qubit-qubit coupling reached at the plateau.
             rise_time: Width of the rising (or falling) action of the trapezoidal pulse.
             padding_time: Symmetric padding around the coupler pulse.
+            q0_detune_MHz: Detuning of the first qubit.
+            q1_detune_MHz: Detuning of the second qubit.
 
         """
         self.hold_time = hold_time
         self.coupling_mhz = coupling_mhz
         self.rise_time = rise_time or cirq.Duration(nanos=8)
         self.padding_time = padding_time or cirq.Duration(nanos=2.5)
+        self.q0_detune_MHz = q0_detune_MHz or 0.0
+        self.q1_detune_MHz = q1_detune_MHz or 0.0
```

Comment:
If removing `Optional` as suggested above, can just assign these:
```suggestion
        self.q0_detune_MHz = q0_detune_MHz
        self.q1_detune_MHz = q1_detune_MHz
```